assets/prototype/domain/eventEditor/containers/mutations/useUpdateTicketMutation.js

```diff
@@ -2,9 +2,8 @@ import { useMutation } from '@apollo/react-hooks';
 import { UPDATE_TICKET } from './tickets';
 import useToaster from '../../../../infrastructure/services/toaster/useToaster';
 
-const useUpdateTicketMutation = () => {
+const useUpdateTicketMutation = ({ id = 0 }) => {
 	const toaster = useToaster();
-	const id = 0;
```

Comment:
should we do this same thing for `assets/prototype/domain/eventEditor/containers/mutations/useCreateTicketMutation.js` ?

Reply:
Ticket id is generated on the server-side, so on `useCreateTicketMutation` init we don't have that available yet, only after the mutation has fired and we received a response from the server that id is pushed into Apollo cache.

Reply:
oh duh.. right of course!